projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -253,9 +322,10 @@
     gap: tokens.$cds-global-space-6;
   }
 
-  .monthpicker,
-  .years {
+  .years,
+  .months {
     flex-wrap: wrap;
+    flex-direction: row;
```

With current implementation `.months` is not `dispaly:flex`, Also there is gap between elements.

```suggestion
    flex-direction: row;
    display:flex;
    gap: tokens.$cds-global-space-5;
```

---

@venkateshr06 please add a comment in which PR this will be fixed?

---

Will create a separate PR on top of this PR merge and add vertical gap changes.

---

JIRA story for tracking [CDE-2552](https://vmw-jira.broadcom.net/browse/CDE-2552)